packages/apps/transfer/.env.example

```diff
@@ -10,6 +10,10 @@ KADENA_API_TTIL=
 # Gas limit for transactions, omit to use default value: 750
 GAS_LIMIT=
 
+# Public & private key used to sign the requests to fund accounts
+FAUCET_PUBLIC_KEY=
+FAUCET_PRIVATE_KEY=
+
```

Comment:
Can you move this down, so that GAS_PRICE succeeds GAS_LIMIT

Reply:
Will do 👌

Reply:
Done